src/styles/blocks/card.css

```diff
@@ -0,0 +1,97 @@
+/* #region card */
+.card {
+  width: 200px;
+  padding: 0 16px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__img {
+  display: block;
+  width: 160px;
+  height: 134px;
+  margin: auto;
+  margin-top: 32px;
+}
+
+.card__title {
+  margin-top: 40px;
+
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+
+  color: var(--text-color-main);
+}
+
+.card__product-code {
+  margin-top: 4px;
+
+  font-size: 10px;
+  line-height: 14px;
+
+  color: var(--text-color-secondary);
+}
+
+.card__rating {
+  margin-top: 16px;
+  display: flex;
+  justify-content: space-between;
+
+  font-size: 10px;
+  line-height: 14px;
+
+  color: var(--text-color-main);
+}
+
+.card__price {
+  margin-top: 24px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.card__price-name {
+  font-size: 12px;
+  line-height: 18px;
+
+  color: var(--text-color-secondary);
+}
+
+.card__price-value {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+
+  color: var(--text-color-main);
+}
+
+.card__button {
+  margin: 16px 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
```

Comment:
it could have been replaced to `text-align: center`

Reply:
if I change it, the text is not in the center. Is it a mistake or a recommendation?